gui/wxpython/startup/locdownload.py

```diff
@@ -429,15 +433,15 @@ class LocationDownloadDialog(wx.Dialog):
     Contains the panel and Cancel button.
     """
 
-    def __init__(self, parent, database, title=_("Location Download")):
+    def __init__(self, parent, database, title=_("Project Download")):
```

Comment:
```suggestion
    def __init__(self, parent, database, title=_("Dataset Download")):
```

Reply:
The main idea is downloading datasets, makes sense.